migration-engine/connectors/sql-migration-connector/src/sql_schema_differ.rs

```diff
@@ -394,7 +439,8 @@ impl<'schema> SqlSchemaDiffer<'schema> {
                                 ColumnTypeChange::SafeCast => sql_migration::ColumnTypeChange::SafeCast,
                                 ColumnTypeChange::RiskyCast => sql_migration::ColumnTypeChange::RiskyCast,
                                 ColumnTypeChange::NotCastable => {
-                                    unreachable!("ColumnTypeChange::NotCastable in redefine_tables")
```

Comment:
ping

Reply:
Update: we just discussed this, and we should have NotCastable here. Note from me: I think ColumnTypeChange::NotCastable doesn't exist, so we'll need to add it and handle it (unreachable in other databases).